src/main/java/it/gov/pagopa/payhub/activities/dto/paymentsreporting/PaymentsReportingDTO.java

```diff
@@ -0,0 +1,46 @@
+package it.gov.pagopa.payhub.activities.dto.paymentsreporting;
+
+import it.gov.pagopa.payhub.activities.dto.IngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.OrganizationDTO;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.sql.Timestamp;
+import java.util.Date;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class PaymentsReportingDTO {
```

Comment:
doesn't this entity have an id field?

Reply:
I started from the legacy structure but @marcomatteuccieng added it in the new one, I will add it